lib/flipper/ui/views/layout.erb

```diff
@@ -16,7 +16,13 @@
     <![endif]-->
   </head>
   <body>
-    <div class="container header">
+    <div class="container">
+      <%- unless Flipper::UI.configuration.environment.nil? -%>
+        <div class="alert alert-<%= Flipper::UI.configuration.environment_banner_color %> text-center font-weight-bold">
+          <%= Flipper::UI.configuration.environment.capitalize %> Environment
```

Comment:
I like your idea of just letting environment be a string. Perhaps you could also change the options to `Flipper::UI.configuration.banner_text` and `Flipper::UI.configuration.banner_class`? That feels a little more generic so people can use it for whatever text and alert class they want. What do you think?

Reply:
I like your idea much better. Having it more generic will be more valuable.